samples/ORKCatalog/ORKCatalog/Tasks/TaskListRow.swift

```diff
@@ -127,6 +128,7 @@ enum TaskListRow: Int, CustomStringConvertible {
                 ]),
             TaskListRowSection(title: "Active Tasks", rows:
                 [
+                    .Wait,
```

Comment:
Since it is not an Active Task, it would be misleading to place it under `Active Tasks`. Let's just keep it in `ORKTest` and remove it from `ORKCatalog` for now.